src/services/member-service.ts

```diff
@@ -1,15 +1,40 @@
+// import { type } from "os";
 import logger from "../utils/logger/logger";
 
-// import SQLService from "./sql-service";
+import SQLService from "./sql-service";
 
 class MemberService {
 	static async isMember(student_id = "", email = "") {
 		logger.info(`student_id=${student_id} email=${email}`);
-
 		// Use SQLService to look up if student is a member.
 		// Either student_id or email may be empty. One is guaranteed to be valid.
 
-		return "Yes";
+		let SQLquery = "SELECT * FROM member ";
```

Comment:
Perhaps consider renaming to 'SQL' to match naming convention in sql-service.js?

Reply:
> Also, I think the table is called 'membership'

I looked in the 'membership' table and there's no data containing the emails or student id's, but the 'member' table seems to work and has the right data.

Reply:
![image](https://user-images.githubusercontent.com/54153434/176590946-e0dd4f15-18e3-4f2d-99c5-a8bf55bced34.png)
The 'membership' entries contain only these fields, so I'm not sure how it would be useful in this case.